accounts.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	// "errors"
+	// "time"
+
+	"errors"
+
+	"github.com/distributeddesigns/currency"
+	// "github.com/op/go-logging"
+)
+
+// Account : State of a particular Account
+type Account struct {
+	Balance currency.Currency
```

Comment:
IMO better to not premptively add things like a Mutex. Add them once we need them.